pkgs/development/python-modules/notifications-android-tv/default.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, python3
+, fetchPypi
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "notifications-android-tv";
+  version = "1.0.0";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-jrKauDkgDyEJOLl25yeGgJDHOirDIPNgQi/z0wVNRD4=";
+  };
+
+  buildInputs = with python3.pkgs; [
+    poetry-core
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    httpx
+  ];
+
+  pythonImportsCheck = [ "notifications_android_tv" ];
+
+  meta = with lib; {
+    description = "Python API for sending notifications to Android/Fire TVs";
+    homepage = "https://pypi.org/project/notifications-android-tv/";
```

With a changelog entry, in the future tools can show this in the PR description and add it to the commit message. Using upstream's primary web presence is more of an optinanted change.

```suggestion
    homepage = "https://github.com/engrbm87/notifications_android_tv";
    changelog = "https://github.com/engrbm87/notifications_android_tv/blob/${version}/CHANGES.rst";
```